README.md

```diff
@@ -0,0 +1,164 @@
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**Math magicians** Bookstore is a simple website that displays a list of books and allows you to:
+
+> Display a list of books
+> Add a book
+> Remove a selected book
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  <li><a href="#">HTML</a></li>
+  <li><a href="#">CSS</a></li>
+    <li><a href="#">React vite</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add books**
+- **Remove books**
+- **Add comments**
+- <p align="right">
+  (<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  ## 💻 Getting Started <a name="getting-started"></a>
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Internet connection.
+- Basic knowledge of Git/Terminal.
+- Visual Studio Code Installed.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Install
+
+Install almost all dependencies.
+This one will install ESLint:
+
+> npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x @babel/plugin-syntax-jsx@7.x @babel/preset-react@7.x @babel/preset-react@7.x
+
+This one will install Stylelint:
+
+> npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project,clone the project from https://github.com/haftamudesta/Math-Magicians
+
+> use the command git checkout component to go to the component dirctory
+> to create react use npm create vite@latest -- --template react
+> to install all raect dependencies use npm install after installing reat
+> to run the code use npm run dev
+
+## 👥 Author <a name="author"></a>
+
+👤 **Haftamu Desta**
+
+- GitHub: [@haftamu](https://github.com/haftamudesta)
+- Twitter: [@DestaHaftamu](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+- LinkedIn: [Haftamu Desta](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [] **Make simple calculation**
+- [] **Read a random math-related quote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```